package.json

```diff
@@ -27,6 +27,12 @@
     "preversion": "yarn test:ci",
     "version": "yarn build && git add -A"
   },
+  "workspaces": [
+    "packages/*"
+  ],
+  "resolutions": {
+    "@reach/auto-id": "0.18.0"
+  },
```

Comment:
What happens to `yarn.lock` if this is removed?

Reply:
There's a mix of `0.17.0` and 0.18.0` found across some of our packages without `"resolutions"` being set. If we comb through our packages that use the older version, we will be able to remove it.